### DISCUSSION TITLE
Надеюсь на понимание , нужна помощь с запуском программы

### DISCUSSION THREAD

**Discussion by Pro100T1gr** [community/community#124853] [Programming Help]
### Body

Нужна помощь с запуском программы, как запустить https://github.com/nehmetekle/Car-Management-System-JavaFX-GUI программу с данной ссылки так чтоб всё работало как надо и без ошибок,буду очень презнателен 

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by ebndev** [comment]
Hi @Pro100T1gr, thanks for participating!

Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting to ensure discussions are adhering to our [Code of Conduct](https://github.com/community/community/blob/920a928ffe3ae4863b64a4ea26917046e11f5c1a/CODE_OF_CONDUCT.md). 

We’ll be locking this post for now, but please feel encouraged to create a new discussion in English if you still need support.
